pkg/diagnostics/report.go

```diff
@@ -102,3 +151,28 @@ func printOverriddenEnvVar(appName string, envVars []corev1.EnvVar) {
 	}
 	fmt.Println("")
 }
+
+// isValidArn is a function to validate the ARN format for an s3 buckets
+func isValidArn(arn *string) bool {
+	return strings.HasPrefix(*arn, "arn:aws:s3::") && len(*arn) > len("arn:aws:s3::")
+}
```

Comment:
I think you could use the built-in AWS SDK check instead of writing it.
When I worked on AWS STS I checked the received ARN of a role: `!arn.IsARN(roleARN)` (you will need to add the import: `"github.com/aws/aws-sdk-go/aws/arn"`), here:

https://github.com/noobaa/noobaa-operator/blob/c6727ce97d1647e47f8cc1deeeff3d2e48e71752/pkg/system/phase2_creating.go#L704-L716

Reply:
Done, Thanks